sdk/boms/azure-spring-cloud-dependencies/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+
+## 2.5.0 (2021-05-20)
+
```

Comment:
I think we could make the changelog of azure-spring-cloud-dependencies aligned with that of other azure spring cloud libraries.

Reply:
For some unknown legancy reasons, there is a changelog of azure spring cloud bom, but not of spring boot, perhaps we could consider in the future that we should delete this or add changelog for spring boot as well?

Reply:
Added.